src/Datadog.Trace.AspNet/Datadog.Trace.AspNet.csproj

```
@@ -6,12 +6,19 @@
     <!-- NuGet -->
     <Version>1.11.1-prerelease</Version>
     <Title>Datadog APM Tracing for ASP.NET</Title>
-    <Description>Provides tracing for ASP.NET web applications.</Description>
+    <Description>
+      OBSOLETE. This package is deprecated and exists only for backwards compatibility.
+      Users who need manual instrumentation should reference the "Datadog.Trace" package.
```

No need for this line. This package was never for manual instrumentation.
```suggestion
      Users who need manual instrumentation should reference the "Datadog.Trace" package.
```